Makefile

```diff
@@ -2,26 +2,30 @@
 
 2BPP   := rgbgfx
 ASM    := rgbasm -E
+LINKER := rgblink
+FIXER  := rgbfix \
+  --color-compatible \
```

Comment:
I like the long arguments, makes it a lot clearer what happens! Would expect this one to be split in a LINKER and a LINKERFLAGS variable though.

Reply:
Good idea.

While I was there, I renamed the binaries to be more in line with C conventions. So there's now `ASM` / `ASFLAGS`, `LD` / `LDFLAGS` and `FX` / `FXFLAGS`.